src/tailwind.js

```diff
@@ -42,80 +36,52 @@ module.exports = {
       },
       colors: {
         // START - redesign
-        brand: {
-          100: "#f7f8fc",
+        primary: {
+          DEFAULT: "#3B8CC5",
+          50: "#F7F8FC",
```

Comment:
researched abit about various color namings conventions, maybe can use this as reference ->https://chir.ag/projects/name-that-color/ i guess

Reply:
ok my bad, don't that `name-that-color` references, i'll update